install.sh

```diff
@@ -1,2 +1,4 @@
 #!/bin/bash
-pip3 install --upgrade --user -e .
\ No newline at end of file
+
+# install jc in development mode
+pip3 install --upgrade --editable .
```

Comment:
is it ok to remove the `--user` option?

Reply:
Ah, I see - I think I had to add to my path to get the `--user` part to work:
```
export PATH="$HOME/.local/bin:$PATH"
```
Ultimately, I think it would be better to describe a more general/generic approach:
- Use something like `pyenv` and/or `venv` to install `jc` into a virtual environment. That way it doesn't mess with your system python or other libraries.
- Install the requirements into the virtual environment
- Then use the `install.sh` and `unistall.sh` files you created. I think with the above we can remove the `--user` flag

Reply:
i have python3 installed long time ago with `brew`. 
on new installs I would use `asdf` or `mise`(formerly known as `rtx`)

I get it working in IntelliJ IDEA

So I don't know how to do the (1) and (2) :/

Reply:
Now that I look at it again, the reason for the `--user` flag is so the console-scripts entry point (i.e the `jc` script that gets installed in the path to run the CLI) does not conflict with other installations of `jc`, including installs via a package manager like `brew`.